### DISCUSSION TITLE
Add section my open pull requests

### DISCUSSION THREAD

**Discussion by mvandermade** [community/community#78700] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Give an overview of open pull requests opened by me.

**Comment by raxod502** [comment, marked as answer]
I think that exists already. Are you looking for https://github.com/pulls?

**Reply by mvandermade** [reply]
Thanks, I did not know that! 
I see there is also an option to ask Copilot this from the feed.

**Reply by raxod502** [reply]
Yes... although heaven knows why. I've complained about the latter part at https://github.com/orgs/community/discussions/147305#discussioncomment-13304725.
